docs/api/classifier/class_prediction_error.rst

```diff
@@ -19,13 +23,39 @@ The class prediction error chart provides a way to quickly understand how good y
     # Perform 80/20 training/test split
     X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.20,
                                                         random_state=42)
+    # Instantiate the classification model and visualizer
+    visualizer = ClassPredictionError(
+        RandomForestClassifier(), classes=classes
+    )
 
-.. code:: python
+    # Fit the training data to the visualizer
+    visualizer.fit(X_train, y_train)
 
-    from sklearn.ensemble import RandomForestClassifier
+    # Evaluate the model on the test data
+    visualizer.score(X_test, y_test)
 
+    # Draw visualization
+    visualizer.poof()
+
```

Comment:
The plot generated each time the docs are built is random and this makes it impossible to write an interpretation since it changes with the plot.  I tried to set the `random_state` for sklearn `make_classification` but that didn't work.  Any suggestions?

Reply:
Ah, too bad that setting the `random_state` didn't work! How about if we swap this dataset with our game dataset? I think it's important to add some interpretation so that folks will know how to use this cool visualizer!

Reply:
I got it to work...I just needed to set the `random_state` of the `RandomForestClassifier`